### ISSUE TITLE
[People Picker] 'X' button is not focusable via keyboard navigation

### ISSUE THREAD

**Issue by somoo88** [microsoft/fluentui#17911]
### Environment Information

- **Package version(s)**:  "@fluentui/react": "^7.129.0"
- **Browser and OS versions**: Edge v89.0.774.57

### Describe the issue:

The whole persona card is focusable, not the remove button in a people picker component.

### Please provide a reproduction of the issue in a codepen:

https://codepen.io/ayeandy/pen/poRaXey

#### Actual behavior:

The focus is on the whole persona card

#### Expected behavior:

The focus should go on the 'x' button on the person card

**Comment by gouttierre** [comment]
@somoo88 - thanks for filing this item and sharing the code pen.

@miroslavstastny - Would you be able to confirm if this is a regression, or if this behavior is an issue?
cc @smhigley
